crates/provider/src/provider.rs

```diff
@@ -995,10 +996,35 @@ impl<N: Network, T: Transport + Clone> Provider<N, T> for RootProvider<N, T> {
     }
 }
 
+/// Admin namespace rpc interface that gives access to several non-standard RPC methods.
+#[cfg_attr(target_arch = "wasm32", async_trait::async_trait(?Send))]
+#[cfg_attr(not(target_arch = "wasm32"), async_trait::async_trait)]
+#[allow(unused, unreachable_pub)]
+pub trait AdminApi {
```

Comment:
how well supported is this namespace? if all clients support it, it should just be a part of the provider trait itself

Reply:
this is only ever enabled locally because privileged access.
An extension trait makes sense here imo